resources/user-management.yaml

```diff
@@ -14,6 +14,24 @@ tags:
   - name: Auth
   - name: Health
 paths:
+  /users:
+    get:
+      tags:
+        - User
+      summary: Gets the detailed account info of users using their ids
+      operationId: getUsersInfo
+      parameters:
+        - $ref: '#/components/parameters/headerCookie'
+        - $ref: '#/components/parameters/queryUserIds'
+      responses:
+        '200':
+          $ref: '#/components/responses/200UsersInfo'
+        '400':
+          $ref: '#/components/responses/400BadRequest'
+        '403':
+          $ref: '#/components/responses/403Forbidden'
+        '404':
```

Comment:
In the implementation the API responds only with status code 200 or 400. If it is correct, please remove the other responses